controllers/front/HistoryController.php

```diff
@@ -82,7 +76,7 @@ public static function getUrlToInvoice($order, $context)
     {
         $url_to_invoice = '';
 
-        if ((bool)Configuration::get('PS_INVOICE') && OrderState::invoiceAvailable($order->current_state) && count($order->getInvoicesCollection())) {
+        if ((bool) Configuration::get('PS_INVOICE') && OrderState::invoiceAvailable($order->current_state) && count($order->getInvoicesCollection())) {
```

Comment:
Yes, it comes from the [Symfony Standards](http://cs.sensiolabs.org/).
You can set you PHP CS Fixer to Symfony standards, it will apply automatically.